src/components/EmptyList.tsx

```diff
@@ -0,0 +1,22 @@
+import { t } from 'i18next'
+import SearchMeta from 'src/types/SearchMeta'
+
+interface IEmptyList {
+  meta?: SearchMeta
+}
+
+export default function EmptyList({ meta }: IEmptyList) {
+  if (!meta) {
```

Comment:
I think make `meta` required and leave it to the parent to handle null case, e.g.

```javascript
{meta && <SearchResultsInfo meta={meta} />}
```